dev.docker-compose.yml

```diff
@@ -22,12 +22,14 @@ services:
       - MARABUNTA_MODE=demo
       - VIRTUAL_HOST=shopinvader-docky.dy
       - VIRTUAL_PORT=8069
+      - DEMO=True
```

Comment:
what about having KEYCHAIN_KEY_DEV defined here so it is explicit it should be customized?